src/functionalTest/java/uk/gov/hmcts/darts/configuration/AccessTokenClientConfiguration.java

```diff
@@ -11,11 +12,13 @@
 @EnableConfigurationProperties
 @RequiredArgsConstructor
 @Profile("functionalTest")
+@Slf4j
 public class AccessTokenClientConfiguration {
 
     private final AzureAdAuthenticationProperties adAuthenticationProperties;
     private final AzureAdB2CAuthenticationProperties b2cAuthenticationProperties;
     private final AzureAdB2CGlobalAuthenticationProperties b2cGlobalAuthenticationProperties;
+    private final AzureAdB2CDarPcMidtierGlobalAuthenticationProperties b2CDarPcMidtierGlobalAuthenticationProperties;
```

Comment:
Should this be b2cDarPcMidtierGlobalAuthenticationProperties to be more like the other b2c fields?